### ISSUE TITLE
Color Considerations for Accessibility

### ISSUE THREAD

**Issue by abhijit945** [cerner/carbon-graphs#15]
Allow for considerations for color contrast for accessibility. Refer to the Draft [Accessibility in Data Viz Doc ]( https://cernerprod-my.sharepoint.com/:w:/g/personal/lb047858_cerner_net/EVJgH70BOuhIpp3auhLe6qUBtNoyeiBwinXiAXKUKqYiBg?e=0tY2NF)for requirements.

**Comment by laurenbock** [comment]
I would like to specifically call out that we need to change the opacity on the hash background bar to a higher percentage because even with black, 30% will not pass accessibility 3:1 contrast.

**Comment by abhijit945** [comment]
@laurenbock Can you create a new issue for this? It'll be easier to track that way. We can always link it here if we want.

**Comment by laurenbock** [comment]
New issue just for background hash opacity update: https://github.com/cerner/carbon-graphs/issues/113
